fx/request.ts

```diff
@@ -9,7 +9,7 @@ export function* request(
   return response;
 }
 
-// deno-lint-ignore no-explicit-any
-export function* json(response: Response): Operation<any> {
-  return yield* call(() => response.json());
+export function* json(response: Response): Operation<unknown> {
+  const result = yield* call(response.json());
+  return result;
```

Comment:
We should keep the `v4` compatible `call()` syntax. Otherwise, it will break in v4

```suggestion
  return yield* call(() => response.json());
```